dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/CoverageMapEndpointServices.java

```diff
@@ -147,35 +141,16 @@ private String validateTokenType(HttpServletRequest request, HttpServletResponse
             throws IOException, OidcServerException {
         String queryString = request.getQueryString();
         if (queryString == null) {
-            String errorMsg = TraceNLS.getFormattedMessage(this.getClass(),
-                    MESSAGE_BUNDLE,
-                    "OAUTH_COVERAGE_MAP_MISSING_PARAMS",
-                    new Object[] { OAuth20Constants.TOKEN_TYPE },
-                    "CWWKS1434E: Missing required parameters in request.");
-            Tr.error(tc, errorMsg);
-
-            throw new OidcServerException(errorMsg, OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
+            throw new OidcServerException(new BrowserAndServerLogMessage(tc, "OAUTH_COVERAGE_MAP_MISSING_PARAMS", new Object[] { OAuth20Constants.TOKEN_TYPE }), OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
Similar comment as above - update this to ensure you're still calling `Tr.error()` to emit the error message in the server logs.